exporter/azure_monitor/example/ocServer.go

```diff
@@ -4,18 +4,29 @@ import (
 	"log"
 	"net/http"
 
+	"go.opencensus.io/exporter/azure_monitor"
+	"go.opencensus.io/exporter/azure_monitor/common"
 	"go.opencensus.io/plugin/ochttp"
+	"go.opencensus.io/trace"
 )
 
 func main() {
 	originalHandler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		w.Write([]byte("Hello, World!"))
+		exporter, err := azure_monitor.NewAzureTraceExporter(common.Options{

```

Comment:
This seems to be in the wrong place. Isn't it suppose to be in the `main` func instead of the handler?

Reply:
Yes, I did not realize that. I am creating a new exporter for every response which is a waste. I moved the creation code out of there.